src/foam/u2/view/UnstyledTableView.js

```diff
@@ -455,7 +455,7 @@ foam.CLASS({
                   }).
                   callIf(view.dblclick && ! view.disableUserSelection, function() {
                     tableRowElement.on('dblclick', function() {
-                      view.dblclick && view.dblclick(canObjBeBuildFromProjection ? obj : null, obj.id);
+                      view.dblclick && view.dblclick(null, obj.id);
```

Comment:
but if canObjBeBuildFromProjection is false, then we have a "full" object here. why do we need to look it up again?

Reply:
we discussed yesterday to always load the object regardless, right? @kgrgreer

Reply:
oh I see. I missed this part